src/ui/server/helpers/mappings/map-applications/index.ts

```diff
@@ -17,7 +17,6 @@ export const mapApplication = (application: Application) => {
     status,
     lastUpdated: formatDate(new Date(updatedAt)),
     referenceNumber,
-    buyerLocation: buyer?.country?.name || DEFAULT.EMPTY,
```

Comment:
I don't think this should be removed and should instead be reverted - this will remove the buyer location column from the dashbaord table?